### PR TITLE
Add EnumFieldListFilter

### LINKED ISSUE #41: Django admin AttributeError 'unicode' object has no attribute 'value'
AttributeError at /admin/someapp/someclass/
'unicode' object has no attribute 'value'
Request Method:	GET
Request URL:	http://127.0.0.1:8000/admin/someapp/someclass/?continent2__exact=Continent.AFRICA
Django Version:	1.8.4
Exception Type:	AttributeError
Exception Value:	
'unicode' object has no attribute 'value'
Exception Location:	/Users/jessamyn/.virtualenvs/mentor/lib/python2.7/site-packages/enumfields/fields.py in get_prep_value, line 35

### PR DESCRIPTION
**Pull request by gavinwahl**
This fixes #41

### CODE REVIEW

File: enumfields/admin.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+from django.contrib.admin.options import IncorrectLookupParameters
+from django.contrib.admin import FieldListFilter
+
+
+class EnumFieldListFilter(FieldListFilter):
+    def __init__(self, field, request, params, model, model_admin, field_path):
+        super().__init__(field, request, params, model, model_admin, field_path)
```
Comment (akx) [inline enumfields/admin.py]: This super call needs to have `EnumFieldListFilter, self` to retain compatibility with Python 2.

**Comment by akx** [comment]
@gavinwahl Could you add the missing super args (and preferably also a test case for this)?

If you don't have the time to do that right now, no biggie -- just say the word and I can do that too.

**Comment by gavinwahl** [comment]
Hey @akx, I'm not able to work on this now so feel free to take over.

Thanks

**Comment by akx** [comment]
Superseded by #56, so closing. Thanks for the initial implementation though, @gavinwahl!

**Pull request closed without merging**